assets/geodata/shapefiles.md

```diff
@@ -0,0 +1,156 @@
+# Shapefiles for the Globe component
+
+## Background
+
+Displaying the in-built geojson supplied with amCharts is problematic for us since it takes positions on disputed territories, whereas we want to be seen to be politically neutral, and provide a product that can be uncomplicatedly used in any of the territories that the underlying model package might support.
```

Comment:
I'll delete the clause for the sake of maximum accuracy

Reply:
```suggestion
Displaying the in-built geojson supplied with amCharts is problematic for us since it takes positions on disputed territories, whereas we want to provide a product that can be uncomplicatedly used in any of the territories that the underlying model package might support.
```